packages/utils/upgrade/src/cli/index.ts

```diff
@@ -13,7 +13,8 @@ const command = (name: string, description: string): Command => {
     .description(description)
     .option('--dry-run', "Run the upgrade, but don't update the files", false)
     .option('-d, --debug', 'Get more logs in debug mode', false)
-    .option('-s, --silent', "Don't log anything", false);
+    .option('-s, --silent', "Don't log anything", false)
+    .option('-p, --project-path <project-path>', 'Path to the Strapi project');
```

Comment:
I think ultimately we should flip this around, and path should be the sole argument, while version should be an --option, but for now this works. As long as we discuss later before releasing anything.